docs/06-concepts/19-experimental.md

```diff
@@ -46,6 +50,37 @@ class ChildClass extends ParentClass {
 }
 ```
 
+### Sealed Classes
+
+In addition to the `extends` keyword, you can also use the `sealed` keyword to create sealed class hierarchies. A sealed class restricts which other classes can extend or implement it, allowing for more controlled inheritance. For example:
```

Comment:
Suggestion: Describe the sealed classes allows exchaustive type checking.

> A sealed class restricts which other classes can extend or implement it, allowing for more controlled inheritance. For example

This description doesn't really apply to us in our model case. Better to describe that it allows for exhaustive type checking. I think that is the most common use-case for sealed classes in models.